docs/rfcs/826-add-multipart-http-signer-support.md

```diff
@@ -0,0 +1,78 @@
+- Author: abyss-w <mad.hatter@foxmail.com>
+- Start Date: 2021-09-26
+- RFC PR: [beyondstorage/go-storage#826](https://github.com/beyondstorage/go-storage/pull/826)
+- Tracking Issue: [beyondstorage/go-storage#0](https://github.com/beyondstorage/go-storage/issues/0)
```

Comment:
Please update the tracking issue link here

Reply:
Done